app/mixins/inplace2.js

```diff
@@ -0,0 +1,58 @@
+import Ember from 'ember';
```

Comment:
`InPlace` mixin is used currently used in the components below: 
- `inplace-boolean`
- `inplace-date`
- `inplace-select`.

`InPlaceValidated` is used in two components now:
- `inplace-text`
- `inplace-textarea`

I will need to see if the code can be applicable to the three components at the top. Sometimes, one-for-all mixin can have issues as it doesn't provide much flexibility.